shell/components/SortableTable/index.vue

```diff
@@ -330,14 +330,24 @@ export default {
     $main.off('scroll', this._onScroll);
   },
 
-  updated() {
-    this.updateLiveAndDelayed();
-  },
-
   watch: {
     eventualSearchQuery: debounce(function(q) {
       this.searchQuery = q;
     }, 200),
+
+    page(neu, old) {
+      if (neu !== old) {
+        this.updateLiveAndDelayed();
+      }
+    },
+
+    pagedRows() {
+      // Ensure we update live and delayed columns on first load
+      if (!this.loading && !this._didinit && this.rows?.length) {
```

Comment:
This line doesn't take in to account the change of `loading` from true to false (on workloads page restarts and health columns just show the loading animation)

Reply:
Fixed